cmd/ferretdb/main.go

```diff
@@ -272,7 +361,20 @@ func setupMetrics(stateProvider *state.Provider) prometheus.Registerer {
 }
 
 // setupLogger setups zap logger.
-func setupLogger(stateProvider *state.Provider, format string) *zap.Logger {
+// It should be preffered for specific single commands/operations over setupLoggerWithStartupMsg.
+func setupLogger(format string, uuid string) *zap.Logger {
```

Comment:
The old `setupLogger` - currently `setupLoggerWithStartupMsg` is used only once as well. The same goes for most of the functions in the main file.

I believe it was done like that, because we initialize a lot of stuff in `run` function, which can affect readability.

I'm not quiet sure if we want to output all of the startup messages such as `"Starting FerretDB"` inside of administration/diagnostic subcommand.
In the same way `docker ps` doesn't return all of the build information.

The second thing is that `ferretdb ping` will be called on each container initialization, so all of the build information will be already logged by `ferretdb` listener itself. If we output another message like that again, the logs will be just confusing.

If we don't want to introduce another function, I propose moving the startup message to the `run` function itself.

Reply:
Thanks for the explanation. Looks great!